blocks/floating-images/floating-images.css

```diff
@@ -0,0 +1,147 @@
+/* styles for the floating images auto block */
+
+.block.floating-images {
+    margin-bottom: 64px;
+}
+
+.block.floating-images > div {
+    display: flex;
+    flex-direction: column-reverse;
+    padding-top: 1em;
+    min-height: 350px;
+}
+
+.section.floating-images-container {
+    padding-left: 24px;
+    border-bottom: 1px solid #979797;
+}
+
+.block.floating-images .image {
+    position: relative;
+    padding: 1.5em 0;
+}
+
+.block.floating-images .image img {
+    object-fit: cover;
+    object-position: center;
+    height: 100%;
+    max-width: 100%;
+}
+
+.block.floating-images .content {
+    border-left: 3px solid #e10;
+    padding: 0 24px;
+    max-height: 350px;
+}
+
+/*** move-to-styles? ***/
+.block.floating-images .content p {
+    margin-bottom: 1.5em;
+    padding: 0 0 16px;
+    line-height: var(--line-height-s);
+    font-size: var(--body-font-size-m);
+    font-weight: var(--font-weight-light);
+    color: var(--quartz);
+}
+
+/*** move-to-styles? ***/
+.block.floating-images .content h2 {
```

Comment:
if this is what *all* h2 will use, yes, we can move this to the global styles file.

Reply:
moved to styles